g2p_programs/views/programs_view.xml

```diff
@@ -197,6 +208,7 @@ Part of OpenG2P. See LICENSE file for full copyright and licensing details.
                                         widget="badge"
                                         optional="show"
                                     />
+                                    <field name="entitlement_ids" widget="many2many_tags" invisible="1" />
```

Comment:
This was already replaced with `entitlements_count` field in the latest merge due to performance issue. Considering that this fixed is to resolve an issue you encountered with spp_dashboard, which refers to the entitlement_ids in xpath to insert a button, the spp_dashboard was also fixed in a separate approved PR.  I suggest that while that PR is not yet merged you uninstall the spp_dashboard in your odoo instance.

Reply:
@gonzalesedwin1123 okay, I will drop this commit